docs/dev-guides/PERSONAL-ENVIRONMENTS.md

```diff
@@ -0,0 +1,523 @@
+created (`terra-jcarlton`). There are numerous examples of the conventions in this file, and I admit
+I cheated a little off the surrounding entries. Note that Atlantis is a tool for managing Terraform updates,
+but atlantis.yaml is not itself a Terraform file.
+
+Second, we specify some workflow settings for deploying the environment. Among other things, this tells
+Atlantis to pass the variables file `jcarlton.tfvars`
+```yaml
+  terra-env-jcarlton:
+    plan:
+      steps:
+        - init:
+            extra_args: ["-upgrade"]
+        - plan:
+            extra_args: ["-var-file", "tfvars/jcarlton.tfvars"]
+```
+
+Third, we check in a tfvars file for Terraform to use as its input variables when building this [workspace](https://www.terraform.io/docs/state/workspaces.html).
+```hcl-terraform
+# General
+google_project = "terra-kernel-k8s"
+cluster        = "integration"
+cluster_short  = "integ"
+
+terra_apps = {
+  workspace_manager = true,
+  crl_janitor       = true,
+  buffer            = true,
+}
+
+# Resource Buffer Service Testing
+buffer_google_folder_ids   = ["637867149294"]
+buffer_billing_account_ids = ["01A82E-CA8A14-367457"]
+
+# Workspace Manager
+wsm_workspace_project_folder_id = "1094831421623" # DSP Integration
+wsm_billing_account_ids         = ["01A82E-CA8A14-367457"]
+```
+This file should be in a path like `terraform-ap-deployments/terra-env/tfvars/jcarlton.tfvars`. IDs will vary, thougth I borrowed
+mine from a teammate. Folder IDs refer to [GCP Project Folders](https://cloud.google.com/resource-manager/docs/creating-managing-folders).
+The entries under `terra_apps` describe services your deployment will depend on (but not deploy). Google project and cluster
+information should be the same for everyone.
+
+The repository is integrated with [Atlantis](https://www.runatlantis.io/), which checks terraform artifacts
+before merge to make sure they can `plan` and `apply` successfully. When pushing a PR to this repo, 
+it's expected you'll run `atlantis` commands as part of the review process. First, a github comment of
+ ```shell script
+ atlantis plan -p terra-env-jcarlton
+```
+to which `broadbot` will reply:
+```text
+Ran Plan for project: terra-env-jcarlton dir: terra-env workspace: jcarlton
+
+Show Output
+```
+
+Following a successful plan, run `atlantis apply`, and if that's successful (and you get an approval), 
+you can merge the PR.
+## Step 2: Edit the Helm File and Auxiliary Files
+[Helm](https://helm.sh/) manages k8s packages called [Helm Charts](https://helm.sh/docs/topics/charts/),
+allowing large-scale Kubernetes deployments. These may be published in public or private repositories,
+similarly to Docker container applications. In turn, a [helmfile](https://github.com/roboll/helmfile)
+may be used to work with many charts in a large, heterogeneous application (like Terra).
+
+### Aside: look through the Charts
```

Comment:
I wanted to be able to say Here's how you build your images, and here's where they're put into pods, then releases, then on down the line. I didn't see how to≠ to talk about the helmfile without at least pointing to helm charts. This his a spot that's really hard to get context on, because you never see the basic k8s files in our workflow. So I can't talk about images or pods really without getting into the helm chart.

Maybe the scope is broader than personal environments, because I wanted to make it more accessible for new people. Maybe another guide to how all the yaml gets merged and interpolated and how to trace it through the system would help?